VideoLocker/src/main/java/org/edx/mobile/discussion/DiscussionTextUtils.java

```diff
@@ -187,7 +186,17 @@ public static CharSequence trim(CharSequence s) {
     }
 
     public static CharSequence removeDuplicateWhitespace(CharSequence text) {
-        return PATTERN_DUPLICATE_WHITESPACE.matcher(text).replaceAll(" ");
+        SpannableStringBuilder builder = new SpannableStringBuilder(text);
+        for (int size = text.length(), i = 0; i < size; i++) {
+            if (Character.isWhitespace(builder.charAt(i))) {
+                int nextIndex = i + 1;
+                if (nextIndex < size && Character.isWhitespace(builder.charAt(nextIndex))) {
+                    builder.replace(i, nextIndex, " ");
+                    break;
```

Comment:
Yeah, we only have the possibility of having dupe spaces after the first span, so, I wrote accordingly.
But lemme refactor it for future use.